resources/css/bem/beatmap-owner-editor.less

```diff
@@ -37,32 +31,23 @@
   &__button {
     .reset-input();
     .link-default();
-    margin-left: 5px;
 
     &[disabled] {
       opacity: 0.5;
     }
   }
 
-  &__input {
-    .default-border-radius();
-    .reset-input();
-    background-color: hsl(var(--hsl-b6));
-    color: inherit;
-    padding: 1px 3px;
-    width: auto;
-    border: 2px solid transparent;
+  &__owners {
```

Comment:
can use some wrapping and more gap between avatar and previous user

![](https://s.kohaku.love/2024/10/firefox_2024-10-02_14-30-22.png)